package.json

```diff
@@ -12,7 +12,7 @@
     "type-check": "sk type-check",
     "build": "sk build",
     "version-bump": "lerna version --no-push --include-merged-tags",
-    "version-bump:checkout": "lerna version --no-push --ignore-changes 'packages/admin-ui-extensions*/**' --include-merged-tags",
+    "version-bump:checkout": "lerna version --no-push --ignore-changes 'packages/{admin-ui-extensions*,ui-extensions}/**' --include-merged-tags",
```

Comment:
Sorry, since you're editing the files can we update both scripts for consistency? Admin needs to include the shared package now
```
"version-bump:checkout": "lerna version --no-push --ignore-changes 'packages/{admin*,ui-extensions}/**' --include-merged-tags",
"version-bump:admin": "lerna version --no-push --ignore-changes 'packages/!({admin*,ui-extensions})/**' --include-merged-tags",
```

Reply:
Good idea. I changed it and tested the command. It produced _No changed packages..._ as expected.